lib/uikit/text/text_extention.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_template/uikit/text/text_style.dart';
+
+/// App text style scheme.
+class AppTextTheme extends ThemeExtension<AppTextTheme> {
+  /// Text style 45/52.
+  final TextStyle display;
+
+  /// Text style 28/36.
+  final TextStyle headline;
+
+  /// Text style 12/16.
+  final TextStyle label;
```

Comment:
I don't think these are valid comments. You can add semantic meaning for these properties, you can look for it in Material specification.